python/lsst/ts/tunablelaser/mock_server.py

```diff
@@ -30,16 +30,43 @@
 ]
 
 import asyncio
+import enum
 import inspect
 import io
 import logging
 import random
+import warnings
 from ipaddress import ip_address
 
 from lsst.ts import tcpip
 
 from .compoway_register import CompoWayFGeneralRegister
-from .enums import Mode, OpticalConfiguration, Output, Power
+from .enums import Mode, Output, Power
+
+# TODO: (DM-46168) Revert workaround for TunableLaser XML changes
+try:
+    from lsst.ts.xml.enums.TunableLaser import OpticalConfiguration
```

Comment:
See my note about this above. Just continue doing what you are doing now and replace the enum once it is available in ts-xml.